WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteViewModel.kt

```diff
@@ -1349,9 +1346,9 @@ class MySiteViewModel @Inject constructor(
         refresh()
     }
 
-    private fun onJetpackFeatureCardLearnMoreClick(url: String) {
+    private fun onJetpackFeatureCardLearnMoreClick() {
         jetpackFeatureCardHelper.track(Stat.REMOVE_FEATURE_CARD_LINK_TAPPED)
-        _onNavigation.value = Event(SiteNavigationAction.OpenJetpackFeatureCardLearnMoreLink(url))
```

Comment:
in commit d0c9e6406043fbe606708dd5eff67e5857be6d2a

Reply:
@AjeshRPai  Not sure about targeting 21.6 since this isn't a breaking change. Plus I con't feel comfortable merging a branch based off of trunk back to 21.6. If it's necc., then I will close this PR/delete the branch and start over from 21.6 - that is safest.